keel-slack/keel-slack.gradle.kts

```diff
@@ -10,6 +10,7 @@ dependencies {
 
   implementation("com.slack.api:slack-api-model-kotlin-extension:1.4.1")
   implementation("com.slack.api:slack-api-client-kotlin-extension:1.4.1")
+  implementation("com.slack.api:bolt-jetty:1.6.0")
```

Comment:
Hmm, we shouldn't need this because we already have a web server (that's what jetty is).

Reply:
fixed